Content.IntegrationTests/Tests/Nyanotrasen/Oracle/OracleTest.cs

```diff
@@ -38,33 +38,36 @@ public async Task AllOracleItemsCanBeTurnedIn()
 
             var testMap = await pairTracker.CreateTestMap();
 
-            await server.WaitAssertion(() =>
-            {
-                var allProtos = oracleSystem.GetAllProtos(oracleComponent);
-                var coordinates = testMap.GridCoords;
+            // This is dumb. All of this is covered by strict checks in OracleSystem and can never happen.
+            // Except for allProtos - but that one also won't happen unless we remove all research altogether.
 
-                Assert.That((allProtos.Count > 0), "Oracle has no valid prototypes!");
-
-                foreach (var proto in allProtos)
-                {
-                    var spawned = entityManager.SpawnEntity(proto, coordinates);
-
-                    Assert.That(entityManager.HasComponent<ItemComponent>(spawned),
-                        $"Oracle can request non-item {proto}");
-
-                    Assert.That(!entityManager.HasComponent<SolutionTransferComponent>(spawned),
-                        $"Oracle can request reagent container {proto} that will conflict with the fountain");
-
-                    Assert.That(!entityManager.HasComponent<MobStateComponent>(spawned),
-                        $"Oracle can request mob {proto} that could potentially have a player-set name.");
-                }
-
-                // Because Server/Client pairs can be re-used between Tests, we
-                // need to clean up anything that might affect other tests,
-                // otherwise this pair cannot be considered clean, and the
-                // CleanReturnAsync call would need to be removed.
-                mapManager.DeleteMap(testMap.MapId);
-            });
+            // await server.WaitAssertion(() =>
+            // {
+            //     var allProtos = oracleSystem.GetAllProtos(oracleComponent);
+            //     var coordinates = testMap.GridCoords;
+            //
+            //     Assert.That((allProtos.Count > 0), "Oracle has no valid prototypes!");
```

Comment:
If this test is no longer necessary at all due to the new way Oracle is coded, then the code for the test should be deleted, rather than commented out. When refactoring, we do not need to keep vestigial remnants of bad code.